src/execution/vm/bytecode_generator.cpp

```diff
@@ -1887,6 +1887,100 @@ void BytecodeGenerator::VisitBuiltinArithmeticCall(ast::CallExpr *call, ast::Bui
   GetExecutionResult()->SetDestination(dest);
 }
 
+void BytecodeGenerator::VisitBuiltinAtomicAndCall(ast::CallExpr *call) {
+  auto args = call->Arguments();
+  LocalVar dest = VisitExpressionForRValue(args[0]);
+  LocalVar val = VisitExpressionForRValue(args[1]);
+  LocalVar ret;
+  auto return_used = GetExecutionResult() != nullptr;
```

Comment:
`auto` isn't more concise than `bool`; I'd prefer the latter. Also, make it `const`.